lizmap/modules/lizmap/lib/Request/Proxy.php

```diff
@@ -275,6 +275,19 @@ protected static function curlProxy($url, $options)
         curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
         curl_setopt($ch, CURLOPT_FOLLOWLOCATION, true);
         curl_setopt($ch, CURLOPT_HTTPHEADER, self::encodeHttpHeaders($options['headers']));
+
+        // For POST, remove parameters from the URL
+        // and add them to $postfields for further use
+        $postfields = '';
+        if ($options['method'] === 'post') {
+            if (empty($options['body'])) {
+                $explode_url = explode('?', $url);
+                if (count($explode_url) == 2) {
+                    $url = $explode_url[0];
+                    $postfields = $explode_url[1];
```

Comment:
I would probably fill `$options['body']` with `$explode_url[1]`, and set the content type to `application/x-www-form-urlencoded` into `$options['headers']`, so no change to do below.